instance/tasks.py

```diff
@@ -339,3 +345,103 @@ def cleanup_old_betatest_users():
+        self.region: str = os.environ.get("KILL_ZOMBIES_REGION", "gra")
+        self.recipient: str = os.environ.get(
+            "KILL_ZOMBIES_WARNING_RECIPIENT", "ops@opencraft.com"
+        )
+        self.threshold: int = int(os.environ.get("KILL_ZOMBIES_THRESHOLD", 10))
+
+    def send_email(self, subject: str, body: str) -> int:
+        """
+        Utility method for sending emails
+        """
+        email = EmailMessage(
+            subject, body, settings.DEFAULT_FROM_EMAIL, [self.recipient]
+        )
+        return email.send()
+
+    def trigger_warning(self):
+        """
+        Warns when more than `threshold` VMs will be terminated
+        """
+        subject = "{} Zombie instances over current threshold ({})".format(
+            datetime.now().strftime("%Y-%m-d"), self.threshold
+        )
+        body = (
+            "The number of instances in region {} to be terminated"
+            "is over KILL_ZOMBIES_WARNING_THRESHOLD ({}).\n"
+        ).format(self.region, self.threshold)
+        logging.info(
+            "%s\nSending an email notification to: %s",
+            body, self.recipient
+        )
+        self.send_email(subject, body)
+
+    def handle_dry_run_output(self, stdout: str) -> int:
+        """
+        If there are servers to terminate, a `kill_zombies` dry run will warn:
+        "Would have terminated {} zombies" to stdout.
+        """
+        if "No servers found in region" in stdout:
+            return 0
+        try:
+            return int(stdout.split("Would have terminated")[1].split()[0])
+        except IndexError:
+            logger.info("Received unexpected input from dry run")
+
+
+@db_periodic_task(
+    crontab(
+        minute=KILL_ZOMBIES_CRON_SCHEDULE[0],
+        hour=KILL_ZOMBIES_CRON_SCHEDULE[1],
+        day=KILL_ZOMBIES_CRON_SCHEDULE[2],
+        month=KILL_ZOMBIES_CRON_SCHEDULE[3],
+        day_of_week=KILL_ZOMBIES_CRON_SCHEDULE[4],
+    )
+)
+def kill_zombies_periodically():
+    """
+    Runs the 'kill_zombies' command to terminate servers
+    whose manual termination didn't succeed.
+    Defaults schedule is daily at 12AM
+    """
+    task_runner = KillZombiesRunner()
+    if task_runner.enabled:
+        logging.info("Executing periodic task `kill_zombies_periodically`")
+        try:
+            dry_run_output, output = "", ""
+            dry_run_output = call_command(
+                "kill_zombies",
+                region=task_runner.region,
+                dry_run=True,
+            )
+            num_of_zombies = task_runner.handle_dry_run_output(dry_run_output)
+            if num_of_zombies > task_runner.threshold:
+                task_runner.trigger_warning()
+            output = call_command("kill_zombies", region=task_runner.region)
+            logging.info(
+                "Task `kill_zombies_periodically`ran succesfully "
+                "and received the following result: %s",
+                output
```

Comment:
@spokerman12, I don't think it is a good idea to clutter the logs with the output of the command. If you still want to log something, it might make sense to just log the number of zombies that were cleaned up.

Reply:
Thank you for suggesting this. Modified as suggested.